tachyon/math/matrix/matrix_utils.h

```diff
@@ -76,6 +79,50 @@ std::vector<PackedPrimeField> PackRowVertically(
   });
 }
 
+// Expands a |RowMajorMatrix|'s rows from |rows| to |rows|^{|added_bits|},
```

Comment:
Shouldn't we add a check if the matrix is row major like PackRowHorizontally?

Reply:
This function became generalized and no longer only pertains to `RowMajorMatrices`. I'll fix the comments!